src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -945,6 +950,22 @@ export class BrightScriptDebugSession extends BaseDebugSession {
         }
     }
 
+    protected async restartRequest(response: DebugProtocol.RestartResponse, args: DebugProtocol.RestartArguments, request?: DebugProtocol.Request) {
+        this.logger.log('[restartRequest] begin');
+        if (this.rokuAdapter) {
+            if (!this.enableDebugProtocol) {
+                this.rokuAdapter.removeAllListeners();
+            }
+            await this.rokuAdapter.destroy();
+            this.rokuAdapterDeferred = defer();
+        }
+        //return to the home screen
+        if (!this.enableDebugProtocol) {
```

Comment:
Do we actually need this? I thought we already press the home button in the normal `launchRequest` flow?